python/smqtk/algorithms/saliency/sbsm.py

```diff
@@ -0,0 +1,266 @@
+from PIL import Image
+import os
+from sklearn.metrics.pairwise import euclidean_distances
+import six
+from smqtk.algorithms import SmqtkAlgorithm
+from smqtk.algorithms.saliency import SaliencyBlackbox,ImageSaliencyAugmenter
+from smqtk.representation.data_element.memory_element import DataMemoryElement
+
+try:
+    import numpy as np
+    from tqdm import tqdm
+    import copy
+except ImportError as ex:
+    logging.getLogger(__name__).warning("Failed to import numpy/tqdm \
+         /copy module: %s", str(ex))
+    np = None
+    tqdm = None
+    copy = None
+
+
+class SBSM_SaliencyBlackbox (SaliencyBlackbox):
+    """
+    SBSM_SaliencyBlackbox function that yields some floating point 
+    scalar value for a given masked base image descriptor element that 
+    signifies the proximity between the query image and masked image
+    descriptors, used by class implementations of 'ImageSaliencyMapGenerator'.
+    """
+
+    def __init__(self, query_f, base_descr):
+        """
+        :param query_f: Feature of query image
+        :param type: smqtk.representation.DescriptorElement
+        :param base_descr: Base image descriptor
+        :param type: smqtk.representation.DescriptorElement
+        """    
+
+        self.query_f = query_f        
+        self.base_descr = base_descr
+
+    @classmethod
+    def is_usable(cls):
+        """
+        Check whether this implementation is available for use.
+        Required valid presence of query image feature 
+        and base image descriptor
+        :return:
+            Boolean determination of whether this implementation is usable.
+        :rtype: bool
+        """
+
+        valid = True
+        if not valid:
```

Comment:
This is now dead code with ``valid = True``. Might as well simply ``return True`` here...